### PR TITLE
Video macro

### PR DESCRIPTION
**Pull request by boxadesign**

### CODE REVIEW

File: src/components/video/_macro.njk [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{%- macro onsVideo(params) -%}
+    <div class="video">
+        {% if params.youtubeUrl %}
+            <iframe class="video__iframe" width="560" height="315" src="{{ params.youtubeUrl }}" frameborder="0" allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (bameyrick) [inline src/components/video/_macro.njk]: Should the width and height on this be hardcoded?

File: src/components/video/examples/video/index.njk [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{% from "components/video/_macro.njk" import onsVideo %}
+{{ 
+    onsVideo({
+        "youtubeUrl": 'https://youtu.be/BiZk2pnSM6w'
```
Comment (bameyrick) [inline src/components/video/examples/video/index.njk]: Youtube short URLs are blocked by most adBlockers, so it's best to use full URLs
```suggestion
        "youtubeUrl": "https://www.youtube.com/watch?v=BiZk2pnSM6w"
```

File: src/components/video/_macro.njk (hunk 1 above)
Comment (boxadesign) [reply src/components/video/_macro.njk]: Well it's what Youtube give you but it's overridden with css.

Comment (bameyrick) [reply src/components/video/_macro.njk]: If it's overridden it's probably best to remove it

**Pull request merged**